### PR TITLE
Avoid too-long filename (LT-18093)

### PR DESCRIPTION
**Pull request by papeh**
Omit blatantly redundant parts of a150-character filename.
TODO in a follow-up: remove further name parts to get a fixed-length
name

---
This change is [Reviewable](https://reviewable.io/reviews/sillsdev/chorus/113)

### CODE REVIEW

**Comment by papeh** [comment]
@palaso-bob, this PR targets `chorus-2.5`, but was built using the `master` configuration, so the checks are invalid.

File: src/LibChorus/VcsDrivers/Mercurial/HgResumeTransport.cs [hunk 1]
```diff
@@ -320,12 +320,8 @@ public void Push()
...
 
 			// create a bundle to push
-			string tip = _repo.GetTip().Number.Hash;
-			string bundleId = "";
-			foreach (var revision in baseRevisions)
-			{
-				bundleId += String.Format("{0}-{1}", revision.Number.Hash, tip);
-			}
+			var tip = _repo.GetTip().Number.Hash;
+			var bundleId = string.Join("_", baseRevisions.Select(rev => rev.Number.Hash)) + '-' +  tip; // REVIEW (Hasso) 2017.02: how much of this is necessary? It's too long when used as a filename. (see also line 639)
```
Comment (jasonleenaylor) [inline src/LibChorus/VcsDrivers/Mercurial/HgResumeTransport.cs]: Does the revision number have a short version of the hash? If so I think we could use that safely. If not LGTM.

**Comment by jasonleenaylor** [comment]
One comment to consider. LGTM

File: src/LibChorus/VcsDrivers/Mercurial/HgResumeTransport.cs (hunk 1 above)
Comment (papeh) [inline src/LibChorus/VcsDrivers/Mercurial/HgResumeTransport.cs]: we could call Substring

**Pull request merged**